cms/djangoapps/contentstore/management/commands/tests/test_create_course.py

```diff
@@ -46,9 +46,6 @@ class TestCreateCourse(ModuleStoreTestCase):
     Unit tests for creating a course in either old mongo or split mongo via command line
     """
 
-    def setUp(self):
-        super(TestCreateCourse, self).setUp(create_user=True)
```

Comment:
It's a violation to not call `super(...).setUp()` somewhere within a `setUp` method. But if you don't have a `setUp` method, then python is just going to call your base-class anyway.